src/mono/System.Private.CoreLib/src/ILLink/ILLink.LinkAttributes.xml

```diff
@@ -149,6 +140,19 @@
     </type>
   </assembly>
 
+  <!-- Attributes that are allowed to be in any assembly -->
```

Comment:
This is true as well for some of the nullable-related attributes as well, e.g. we build AllowNull into assemblies targeting netstandard.  Should those be moved, too?

Reply:
Thanks. I moved the nullable-related attributes as well.